### ISSUE TITLE
0.5.8 Build failure

### ISSUE THREAD

**Issue by yochananmarqos** [BitPonyLLC/huekeys#22]
Building 0.5.8 fails in a clean chroot using my AUR [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=huekeys):

```
curl -sS https://raw.githubusercontent.com/meodai/color-names/master/src/colornames.csv | \
  awk -F, '$1 ~ /^[a-zA-Z]+$/' | \
  gzip > pkg/keyboard/colornames.csv.gz
rm -f huekeys # to make it obvious when watch fails
go generate ./...
make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
make[1]: Entering directory '/build/huekeys/src/huekeys'
echo "$BUILDINFO" | tee buildinfo/build.yml
build_time: 2022-07-04T14:59:27Z
commit_hash: 24e1276
version: v0.5.8
make[1]: Leaving directory '/build/huekeys/src/huekeys'
go build -ldflags='-s -w' -o huekeys
14:59:34.547 WRN unable to use syslog: switched to stderr error="Unix syslog delivery error"
Error: unable to open /build/.local/share/applications/huekeys.desktop: open /build/.local/share/applications/huekeys.desktop: no such file or directory
14:59:34.548 ERR command failed error="unable to open /build/.local/share/applications/huekeys.desktop: open /build/.local/share/applications/huekeys.desktop: no such file or directory"
```

**Comment by bradrf** [comment]
thanks for letting me know. should be fixed in v0.5.9

**Comment by yochananmarqos** [comment]
EDIT: Redacted. See #23

**Comment by bradrf** [comment]
should be fixed in v0.5.10
